CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+Contribute to Scrap Packed Explorer
+=====================
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow][githubflow], So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow][githubflow]). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/romibi/Scrap-Packed-Explorer/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/romibi/Scrap-Packed-Explorer/issues/new); it's that easy!
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+
+### Naming Conventions
+* Every class and method should be named using **PascalCase** convention (example: `void ReadFile()`)
+* Every method argument should be named using **camelCase** convetion with prefix _p__ (example: `string p_filePath`)
+* Every variable should be named using **camelCase** convention (example: `int maxLength`)
+* Every application's command line argument should be named using **camelCase** convention (example: `--keepBackup`)
+<!-- TODO: change args to kebab-case? (--keep-backup) -->
```

Comment:
do we do this?

Reply:
This is only my proposal. Yours word is last

Reply:
hm … i kinda prefer camelCase to read
but maybe allow `--verb-subject` as an exception
to also allow `--show-fileSize` and `--match-fileName` and similar
so camelCase with one dash allowed?
what do you think?

but when parsing it should be case-insensitive then
if it can't be case insensitive, yes then i think from a usability point we should use kebab-case …

Reply:
It can be case-insensetive

Reply:
via some parser rule or by converting arguments to lowercase?
because … if i define the filename for saving stuff with upper case letters i want them to be upper case letters …

Reply:
Via parser rule. Applies only to options, not to values